[Bug] Typo in critic model name: 'gpt-4o-mni' instead of 'gpt-4o-mini'

The critic agent config in orchestrator.py has model 'gpt-4o-mni' (typo — letters 'i' and 'n' swapped).

This causes an OpenAI API NotFoundError when the critic agent tries to respond during debates.

Expected: `gpt-4o-mini`
Actual: `gpt-4o-mni`

File: `src/glassbox/orchestrator.py`, line 12

🧪 **GlassBox Tester**

| Check | Result |
|-------|--------|
| 📐 TP1 Syntax | ✅ OK |
| 🧪 TP2 Full suite | ❌ 6 failed / 52 total |
| 📏 TP3 Diff size | ✅ 1 lines (max 3) |

**Manager's edge cases verified:**
- ⚠️ 🟢 T1: `Correct model name 'gpt-4o-mini' is used.` → The system should function without errors and the critic agent should respond correctly.
- ⚠️ 🟡 T2: `Model name with different typo, e.g., 'gpt-4o-min'` → The system should raise an OpenAI API NotFoundError.
- ⚠️ 🟠 T3: `Model name is completely incorrect, e.g., 'gpt-xyz'` → The system should raise an OpenAI API NotFoundError and log the error.
- ⚠️ 🔴 T4: `Model name is empty.` → The system should raise an error indicating that the model name cannot be empty.

**Failures:**
- `test_15_personas_have_required_keys`: AssertionError: model missing fields
- `test_16_personas_temperature_in_range`: TypeError: '<=' not supported between instances of 'int' and 'str'
- `test_17_orchestrator_filters_invalid_agents`: AssertionError: assert ['architect'] == ['architect', 'critic']
- `test_18_orchestrator_default_agents_match_personas`: AssertionError: assert ['architect',...ist', 'model'] == ['architect',...st', 'critic']
- `test_22_prompts_updated_to_design_review`: AssertionError: model prompt not updated

❌ Fix needs work. 🔧 **Junior Dev**, please retry.

--- a/src/glassbox/orchestrator.py
+++ b/src/glassbox/orchestrator.py
@@ -9,7 +9,7 @@ from .trust_db import TrustDB
 AGENTS = {
     "architect": ("gpt-4o", 0.3, "You are @architect. Think long-term, scalability, what breaks at scale. Talk like you're in a design review — direct, opinionated, no fluff. Reference @pragmatist/@critic by name. Agree or disagree sharply."),
     "pragmatist": ("gpt-4o", 0.5, "You are @pragmatist. Ship fast, iterate, cut scope. Talk like you're in a design review — direct, opinionated, no fluff. Reference @architect/@critic by name. Push back on overengineering."),
-    "critic":     ("gpt-4o-mni", 0.4, "You are @critic. Find edge cases, failure modes, security holes. Talk like you're in a design review — direct, opinionated, no fluff. Reference @architect/@pragmatist by name. Challenge assumptions."),
+    "critic":     ("gpt-4o-mini", 0.4, "You are @critic. Find edge cases, failure modes, security holes. Talk like you're in a design review — direct, opinionated, no fluff. Reference @architect/@pragmatist by name. Challenge assumptions."),
 }
 ROUNDS = [
     "ROUND 1: State your position. Be direct, no bullet points.",

